dm2-assignment-09.3.org

```diff
@@ -587,8 +607,7 @@ NOTES FROM the Textbook:
   ;; Return T (Q's first and only element?)
 
   (defun sym-greater-than (n1 n2)
-    (and (symbolp n1) (symbolp n2)
-      (< (get-value n1) (get-value n2))))
+    (> (get n1 :frequency) (get n2 :frequency)))
```

Comment:
Is this how you got the sort to work correctly? This way the tree is built with the most frequent values first and avoid trying to sort cons cells. Could something like <code>(sort Q '> ':frequency)</code> work for this instead of calling a function in the make-huffman-tree down below?

Reply:
Nevermind, I think you would still have to make the :frequency part a lambda expression to get it to work this way. Making a separate function is more readable than that. Looks good to me 👍